maintainers/maintainer-list.nix

```diff
@@ -1036,6 +1036,12 @@
     githubId = 1015044;
     name = "Brandon Carrell";
   };
+  bcc32 = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add bcc32``.